pkg/scheduler/objects/node.go

```diff
@@ -72,9 +81,9 @@ func NewNode(proto *si.NodeInfo) *Node {
 		allocations:       make(map[string]*Allocation),
 		schedulable:       true,
 		listeners:         make([]NodeListener, 0),
+		ready:             ready,
```

Comment:
On this abstraction level, this field could have a different name like `acceptingContainers` or just `healthy`.